### PR TITLE
Fix typos (.:: should typically just be ::).

### PR DESCRIPTION
**Pull request by JulienPalard**
Found them using:

    git grep "\.::" Doc/ | grep -v 'e\.g\.::'

### CODE REVIEW

File: Doc/faq/windows.rst [hunk 1]
```diff
@@ -60,7 +60,7 @@ program. So, how do you arrange for the interpreter to handle your Python?
 First, you need to make sure that your command window recognises the word
 "python" as an instruction to start the interpreter.  If you have opened a
 command window, you should try entering the command ``python`` and hitting
-return.::
+return::
```
Comment (serhiy-storchaka) [inline Doc/faq/windows.rst:63]: This should be `. ::` (needed a space between `.` and `::`).

**Review by serhiy-storchaka — COMMENTED** [review]
I think it should be rather `. ::`.

File: Doc/faq/windows.rst (hunk 1 above)
Comment (pitrou) [reply Doc/faq/windows.rst:63]: `::` gets displayed as `:`, so it's wrong to add a point before it.

**Review by pitrou — APPROVED** [review]

File: Doc/faq/windows.rst (hunk 1 above)
Comment (serhiy-storchaka) [reply Doc/faq/windows.rst:63]: `::` after a space is not displayed, but just starts a code block. I believe this was the initial intention of the author of this text. See other occurrences of `. ::` in the documentation.

**Review by serhiy-storchaka — DISMISSED** [review]

File: Doc/faq/windows.rst (hunk 1 above)
Comment (JulienPalard) [reply Doc/faq/windows.rst:63]: The following block is clearly the example of the explained sentence. Sentence is "entering the command ``python`` and hitting return", bloc is "C:\Users\YourName> python", so ":" make more sense than "." here, am I wrong?

**Comment by serhiy-storchaka** [comment]
Do you mind to create a backport to 2.7 @JulienPalard?

**Pull request merged**